package.json

```diff
@@ -80,7 +80,7 @@
     "eslint-plugin-react": "^7.20.2",
     "eslint-plugin-react-hooks": "^4.0.4",
     "fork-ts-checker-webpack-plugin": "^5.0.4",
-    "jest": "^26.4.2",
```

Comment:
Why was this change made?

Reply:
We are not sure about all the changes in this file. We'll look into it